src/mumble/Database.cpp

```diff
@@ -220,6 +220,8 @@ Database::Database(const QString &dbname) {
 							 "`server_cert_digest` TEXT NOT NULL, `channel_id` INTEGER NOT NULL)"));
 	execQueryAndLogFailure(query, QLatin1String("CREATE UNIQUE INDEX IF NOT EXISTS `filtered_channels_entry` ON "
 												"`filtered_channels`(`server_cert_digest`, `channel_id`)"));
+	query.exec(QLatin1String("ALTER TABLE `filtered_channels` ADD COLUMN `filter_mode` INTEGER DEFAULT ")
+			   + QString::number(Channel::FILTER_MODE_HIDE)); // Upgrade path, failing this query is not noteworthy
```

Comment:
I'm not actually sure that the default value will be applied to existing rows - they might end up with `NULL` entries in the new column. Did you test this upgrade path?

Reply:
I just confirmed that the upgrade path works as intended at least with sqlite. (This was, by the way, also implemented like this by @hacst)